vala-libsoup.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name" : "vala-mod",
```

Comment:
Why not use the vala extension here https://github.com/flathub/org.freedesktop.Sdk.Extension.vala ?

Reply:
I used it at the beginning, but because of the error I decided to limit the possibilities of failure. I decided to keep this arrangement because it is lighter.